src/App.css

```diff
@@ -1,51 +1,142 @@
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  background-color: rgba(0, 0, 0, 0.05);
+  font-family: 'Montserrat', sans-serif;
+}
+
+.header {
+  text-transform: uppercase;
+  margin: 8rem 0 0 2rem;
+  font-size: 2rem;
+  color: #888;
+  font-weight: bold;
+}
+
+button {
+  padding: 1rem;
+  background-color: #0290ff;
+  width: 5rem;
+  height: 3rem;
+  color: #fff;
+  font-family: Montserrat, sans-serif;
+  font-size: large;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
```

Comment:
- [ ] As requested by the previous reviewer, it would be nicer to remove the border from the buttons to improve your UI design. I would recommend adding `border: none;` to the button selector.

<img width="827" alt="image" src="https://user-images.githubusercontent.com/77190203/162591103-1ab53039-ef83-49f8-a4c6-532e552c9ec1.png">